src/test/java/org/arl/fjage/test/auth/SimpleFirewallSupplier.java

```diff
@@ -0,0 +1,129 @@
+/******************************************************************************
+
+Copyright (c) 2013, Mandar Chitre
+
+This file is part of fjage which is released under Simplified BSD License.
+See file LICENSE.txt or go to http://www.opensource.org/licenses/BSD-3-Clause
+for full license details.
+
+******************************************************************************/
+
+package org.arl.fjage.test.auth;
+
+import org.arl.fjage.AgentID;
+import org.arl.fjage.auth.Firewall;
+import org.arl.fjage.remote.Action;
+import org.arl.fjage.remote.JsonMessage;
+
+import java.util.*;
+import java.util.function.Consumer;
+import java.util.function.Supplier;
+
+public class SimpleFirewallSupplier
+    implements Supplier<Firewall> {
```

Comment:
@mchitre `Firewall` as it stands is pretty low-level. We might we to consider including something like `SimpleFirewallSupplier` that addresses common usage. If you agree, we will need to review this class.

Reply:
Yes, sure, having a basic implementation that is useful is nice. We might want to clearly articulate what the implementation is meant to do and not, and if someone had to extend it to link with authentication systems, what it would take.

Reply:
Might it be better to have just this implementation as a separate PR? Merge the framework first, and then add in a simple implementation?